### PR TITLE
PyYaml: uncomment commented out imports and add missing classmethod decorators

### PR DESCRIPTION
**Pull request by rytilahti**
* tests/pytype_test.py (py2 as instructed in README.md) and tests/mypy_test.py and tests/mypy_selftest.py with python3.6.1 run successfully.
* Tested against https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/util/yaml.py

### CODE REVIEW

File: third_party/2and3/yaml/__init__.pyi [hunk 1]
```diff
@@ -1,10 +1,10 @@
 from typing import Any
-# from yaml.error import *
-# from yaml.tokens import *
-# from yaml.events import *
-# from yaml.nodes import *
-# from yaml.loader import *
-# from yaml.dumper import *
+from yaml.error import *
```
Comment (JelleZijlstra) [inline third_party/2and3/yaml/__init__.pyi]: try `from .error import *` here (I haven't tested it though)

Comment (rytilahti) [inline third_party/2and3/yaml/__init__.pyi]: Unfortunately that doesn't help, from the sound of it flake8 doesn't like wildcard imports. Other ideas?

**Comment by graingert** [comment]
@JelleZijlstra clearly flake8 should ignore `from yaml.error import *` in pyi files.

**Comment by rytilahti** [comment]
Added an ignore for those wildcards.

**Comment by rytilahti** [comment]
Shouldn't that be a separate PR, though?

**Comment by graingert** [comment]
Maybe a pr to flake8 so it ignores import * in pyi files?

**Comment by JelleZijlstra** [comment]
That's not necessarily correct; I'd still want to disallow `from typing import *` in stubs.

**Comment by rytilahti** [comment]
Thanks for the merge! I didn't notice it before and came here to see how to proceed :)

**Pull request merged**